src/lib/components/Cytoscape.react.js

```diff
@@ -676,7 +787,50 @@ Cytoscape.propTypes = {
          */
         boundingBox: PropTypes.object,
     }),
-
+    /**
+     * Define a custom context menu
```

Comment:
In this docstring you should explain how `onClick`, `onClickCustom`, and `contextMenuData` work. (i.e. the three ways of triggering code to run when you click on an item in the context menu.)